browser/include/content_analysis/sdk/analysis_client.h

```diff
@@ -8,7 +8,7 @@
 #include <memory>
 #include <string>
 
-#include "content_analysis/sdk/analysis_agent.pb.h"
+#include "content_analysis/sdk/analysis.pb.h"
```

Comment:
Let's fix this in previous PR to make sure all commits can build successfully.

Reply:
Done.